docs/content/docs/messages-router.md

```diff
@@ -75,32 +71,33 @@ To run the Router, you need to call `Run()`.
 
 #### Ensuring that the Router is running
 
-You might sometimes want to do something after the router starts. You can use the `Running()` method for this.
+It can be useful to know if the router is running. You can use the `Running()` method for this.
 
 {{% render-md %}}
 {{% load-snippet-partial file="content/src-link/message/router.go" first_line_contains="// Running" last_line_contains="func (r *Router) Running()" padding_after="0" %}}
 {{% /render-md %}}
 
 ### Execution model
 
-Some *Consumers* may support only a single stream of messages - this means that until a `msg.Ack()` is sent, you will not receive any more messages.
+*Subscribers* can consume either one message at a time or multiple messages in parallel. Single stream of messages is the simplest approach
+and it means that until a `msg.Ack()` is called, the subscriber will not receive any more messages.
```

Comment:
> and it means that until a `msg.Ack()` is called, the subscriber will not receive any more messages.
it's not so simple, unfortunately :( 

for example, if you are consuming multiple partitions in Kafka, you will receive multiple messages in parallel, event without ack - so, in general, it depends on pub/sub implementation

Reply:
ah ok, it's below, maybe we can rephrase it a bit? because until you don't go below its not clear

Reply:
I was also thinking about a simple schema of how it's working, but maybe its overkill?

Reply:
See 6b1b00e6377f7292d59ad1ab6828b5ef770f3db0

A schema could be nice, although I don't have an idea at the moment how to present this clearly. But the refined docs should be now pretty easy to understand.

Reply:
So maybe just an example how to add (for example different UUID format BIN(16) or something like that)

Reply:
Added general instruction in fe30835c840bd8be368e599cdb8bf7dd3e3c20aa
The example extended code is shown within the included snippet.

I think sometime in the future we can add yet another SQL example, showing how to prepare full schema and offset adapters.